reports/historical/historical.py

```diff
@@ -16,13 +16,20 @@ class Historical(object):
 
     def __init__(self, historical_file="historical.json"):
         self.fname = historical_file
-        self.all_data = {}
+        self.load()
+
+    def load(self):
+        if os.path.exists(self.fname):
+            with open(self.fname, 'r') as f:
+                self.all_data = json.load(f)
+        else:
+            self.all_data = {}
```

Comment:
@esirK , @ascii-dev removed this because he was worried it *could* happen that two tabs in 2010/2015/2020/etc. sheets have the same name and if we don't start with empty `historical` when **importing**, it may lead to situation where you're loading 2015 data but you end up with some of the 2010 data since that was the data loaded the last time the command was invoked.

However, it looks like `report_builder` needs this functionality. Is it possible to call it explicitly after initiating the `historical` instance i.e. `historical.load()`?

Reply:
Makes sense.